src/v2/view-builder/views/IdentifierView.js

```diff
@@ -94,6 +94,13 @@ const Body = BaseForm.extend({
     }
   },
 
+  postRender() {
+    BaseForm.prototype.postRender.apply(this, arguments);
+    // When a user enters invalid credentials, /introspect returns an error,
+    // along with a user object containing the identifier entered by the user.
+    this.$el.find('.identifier-container').remove();
```

Comment:
This is the UX I'm preventing:
![Screen Shot 2022-01-21 at 10 13 10 AM](https://user-images.githubusercontent.com/71431120/150598335-bd7865cc-f2ea-4758-92e5-8a5baa5e63ed.png)

Reply:
good catch!